src/misclibs.F

```diff
@@ -3551,18 +3561,26 @@ subroutine calcksmax(nstat,rstat,dt,uh,vh,mf,kmh,kmv,khh,khv)
       imaxv=1
       jmaxv=1
       kmaxv=1
-      !$acc parallel loop gang vector default(present)
+      !$acc parallel loop default(present)
```

Comment:
Change it to `loop seq` for the same reason above.

Reply:
Thanks @sjsprecious , I made this change.